charter.md

```diff
@@ -53,15 +53,207 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+  - Are there scenarios where unanimity should be considered?
+  - [Stephen] What about a combination of both?
+
+-->
+
+In the course of the committee's operations, members will be expected to vote
+on decisions within the body's purview.
+
+These votes may proceed on agreed-upon platforms by the committee, such as:
+
+* a pull request
+* an issue
+* a Steering Committee [meeting](#meetings)
+* a mailing list
+
+Unless otherwise specified by a process, the requirement for passing a vote is
+a **_two-thirds supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* 5 members in attendance would be required to vote during a meeting
+
+### Abstention
+
+For any self-elected reason, members of the committee may decide to abstain
+from a vote.
+
+Abstaining members will only be considered as contributing to quorum, in the
+event that a vote is called in a meeting.
+
+## Meetings
+
+Steering Committee members are generally expected to attend every meeting. We
 use the following guidelines to determine whether we have reached quorum and
 are able to proceed with a meeting.
 
-* Round quorum up if total number of members is odd
-* Quorum to meet is 1/2 members attending (eg: 6/12, 6/11, 4/7)
-* Quorum to vote in a meeting is 2/3 members attending (eg: 8/12, 8/11, 5/7)
+### Quorum
+
+Quorum **to meet** is a **_majority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* 4 members in attendance would be required to meet
+
+Quorum **to vote in a meeting** is a **_two-thirds supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* 5 members in attendance would be required to vote during a meeting
+
+## Elections
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+SIG Contributor Experience may delegate this responsibility to a subproject.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+Policy and operations of special elections initiated by the Steering Committee
+for other reasons (as yet to be determined) may be handled by some agreed-upon
+[process][changes] by the committee.
+
+[general-elections]: /elections.md
+[sig-contribex]: https://git.k8s.io/community/sig-contributor-experience
+
+### Vacancies
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
```

Comment:
I would set a limit on how many people can be taken up this way, otherwise eventually you get to candidates who the community has basically completely rejected. Maybe something like no more than two people can be obtained this way, and anyone obtained this way must also be in the top 25% of that election.

Reply:
oh, I see you moved this text. Well, maybe this isn't the best time to fix this (and my subsequent comment).

Reply:
Well, actually, I do think this needs to change given the board can eject people. We do not want the board to go down the list ejecting people until they get the person they like.

Reply:
Maybe only one seat can be filled from the list of candidates based on their votes, if they are then ejected an election must happen, that way it allows for quick/easy resolution of a vacancy, but heavily disincentivizes a Nixon/Watergate situation.